ui/js/controllers/perf/compare.js

```diff
@@ -413,15 +413,14 @@ perf.controller('CompareSubtestResultsCtrl', [
             $scope.compareResults = {};
             $scope.titles = {};
 
-            $scope.subtestTitle = ($scope.platformList[0].split(' ')[0] + " " +
-                                   $scope.testList[0].split(' ')[0]);
-            window.document.title = $scope.subtestTitle + " subtest comparison";
```

Comment:
Yes, on the page header itself, for example is: "linux64: basic_compositor_video pgo subtest summary". I can add 'subtest comparison' or 'summary' back to the window title too if you like!